installer/roles/ansible-service-broker-setup/tasks/main.yml

```diff
@@ -52,6 +56,7 @@
   - dockerhub_org != ''
   - launch_apb_on_bind is defined
   - launch_apb_on_bind != ''
+  - "'ansible-service-broker' not in all_projects.stdout"
```

Comment:
Perhaps here we could check for the exact project and user we're creating. From what I understand at the moment even if we had a user named `test-admin` created it would still skip creating the user?

Instead maybe doing like `oc get project ansible-service-broker` and `oc get user admin` and then checking the return code to determine whether to continue or skip would be more explicit.

What do you think?